packages/drivers/odsp-driver/src/odspDocumentService.ts

```diff
@@ -338,6 +338,7 @@ export class OdspDocumentService implements IDocumentService {
                     nonAfdUrl,
                     this.logger,
                     60000,
+                    this.epochTracker.fluidEpoch,
```

Comment:
This might not be enough in future. I believe it works today because
- application always starts container paused state and we learn about epoch before connecting
- joinSession should bring us epoch knowledge.
But I think when these two conditions are not here (for example, we eventually remove joinSession call - there is no such thing in FRS) and we start container unpaused, then we will not know epoch.

The way it works in other places is that in addition to telling our epoch if we have it, we also learn and validate epoch from API call. I.e. we need to get epoch from PUSH and
- compare it against one we learn from other API (getLatest) if it completed sooner
- remember epoch and use it to validate calls that come after (if getLatest comes later), and fail container if there is epoch mismatch

Reply:
Note that if joinSession does not return epoch (I think I heard that), then we just can't punt on that - we need to fix it before merging this PR, I think. It's just too risky to assume on paused start - this can change at any moment independently.

Reply:
connect_document_success returns epoch in response. So it can be initialized from there and we can make sure that if epoch changes we don't need to rely on tress/latest call and we can just pass the old epoch and get that validated at server and client.